versions/ufo3/glyphs/glif.md

```diff
@@ -343,16 +343,38 @@ The following is a registry of public lib keys that map to functionality that is
 
 This key is used for representing the "mark" color seen in various font editors. The value for the key must be a string following the [color definition] standard. This data is optional.
 
-#### public.truetype.instructions
+##### public.truetype.instructions
 
 This key provides a dict defining a set of TrueType instructions assembly code for a glyph. This data is optional.
 
-#### TrueType Instructions Dict
-| key           | value type | description                                                                                                                                                                                                                                                                                                                                                |
-|---------------|------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| formatVersion | string     | Format version. Set to "1".                                                                                                                                                                                                                                                                                                                                |
-| id            | string     | Hash of glyph outlines which may have been processed by authoring tools. This is computed when the instructions dict is created or modified. It is used to determine if the glyph outlines have changed since the glyph was hinted: if it has, then the instructions for the glyph should not be used by authoring tools. See "Hint ID Computation" below. |
-| assembly      | string     | TrueType instructions assembly. The assembly is as represented with fontTools.                                                                                                                                                                                                                                                                             |
+```xml
+<key>public.truetype.instructions</key>
+<dict>
+  <key>formatVersion</key>
+  <string>1</string>
+  <key>id</key>
+  <string>d0b9efba65108827703ed3e672c5041983cee80e7be6144acc681627361b3929b32e06e6c52b2fee09175a182b8cce21bbf72f145c7969bceabf8e309cf3c2f8</string>
+  <key>assembly</key>
+  <string>
+SVTCA[0]> /* SetFPVectorToAxis */
```

Comment:
those `>` at the end of each instructions should not be there, i think?

```suggestion
SVTCA[0] /* SetFPVectorToAxis */
```

Reply:
Yes, indeed 🤕